eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -37,6 +37,9 @@
     <UsagePattern IdentityGlob="System.Reflection.Metadata/8.0.0" />
     <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/*7.0.0*" />
 
+    <!-- Transitive dependencies from MSBuild -->
+	  <UsagePattern IdentityGlob="Microsoft.VisualStudio.SolutionPersistence/1.0.9" />
```

Comment:
This should not be defined. If a prebuilt is occurring, it means that the necessary version of source-build-externals is not being referenced. This was only merged to the main branch: https://github.com/dotnet/source-build-externals/pull/392. That doesn't flow to 9.0.2xx. It will need to be backported to release/9.0.

Reply:
Actually, I wasn't thinking. This is for 9.0.2xx which isn't source built. So it should be ok to allow this as a prebuilt. Is that ok, @MichaelSimons?

Reply:
It would be fine to add as an allowed prebuilt but I would not want this backported to main.  In main we want to make sure this is handled correctly.